projects/cashmere-examples/src/lib/tabs-horizontal/tabs-horizontal-example.html

```diff
@@ -1,18 +1,18 @@
 <div class="tab-demo">
     <hc-tab-set direction="horizontal">
-        <hc-tab tabTitle="Tab 1">
+        <hc-tab tabTitle='Pending Tasks'>
             <div class="tab-content">
-                Tab content 1
+                This is where pending tasks would go.
             </div>
         </hc-tab>
-        <hc-tab tabTitle="Tab 2">
+        <hc-tab tabTitle='<i class="fa fa-check" style="color: #00a859; font-size: 20px;"></i>  Complete Tasks'>
```

Comment:
It seems strange to me that HTML would be inside an element attribute.  Usually when this kind of thing is needed you just have an optional subelement, i.e.

```suggestion
<hc-tab tabTitle="Pending Tasks">
   <hc-tab-content>
      This is where pending tasks would go.
   </hc-tab-content>
</hc-tab>
<hc-tab>
   <hc-tab-title>
      <i class="fa fa-check" class="complete-tasks"></i> Complete Tasks
   </hc-tab-title>
   <hc-tab-content>
      These are completed tasks
   </hc-tab-content>
</hc-tab>
```